iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -716,7 +725,7 @@ public void execute(IotHubConnectionStatus status, IotHubConnectionStatusChangeR
         this.internalClient.registerConnectionStatusChangeCallback(connectionStatusUpdateCallback, null);
     }
 
-    protected void testGetDeviceTwin() throws IOException, InterruptedException, IotHubException
+    protected void testGetDeviceTwin() throws IOException, InterruptedException, IotHubException, GeneralSecurityException, ModuleClientException, URISyntaxException
```

Comment:
When do we see this one - GeneralSecurityException ?

Reply:
When something goes wrong when creating the SSL context